athlinks-schema.ttl

```diff
@@ -66,13 +73,20 @@ athlinks:AthleticRace
   rdfs:comment "An athletic race, a distinct course." ;
   rdfs:label "Athletic Race" ;
   rdfs:hasProperty athlinks:raceType ;
-  rdfs:subClassOf schema:SportsEvent .
+  rdfs:subClassOf schema:SportsEvent ;
+  rdfs:hasProperty athlinks:giveaway  ;
+  rdfs:hasProperty athlinks:require_phone ;
+  rdfs:hasProperty athlinks:require_dob ;
+  rdfs:hasProperty athlinks:volunteer ;
```

Comment:
all 4 of these... (require_phone, require_dob, volunteer, giveaway) seem out of context.. I'm guessing the should be on the registration setup, or part of collecting registration data? I just don't even know how we would use these properties on the Race... unless volunteer has a range of Person? But what does require_phone mean on a race?